clang/lib/Driver/ToolChains/PS4CPU.cpp

```diff
@@ -141,6 +141,9 @@ void tools::PS4cpu::Linker::ConstructJob(Compilation &C, const JobAction &JA,
   if (Args.hasArg(options::OPT_pie))
     CmdArgs.push_back("-pie");
 
+
```

Comment:
We don't use double blank lines as a logical separator.

Reply:
Oops - thanks. Removed.